### PR TITLE
Release v3.1.0

### PR DESCRIPTION
**Pull request by AndreasVerden**
- [x] Refactor AWS Entry Point
- [x] Update Dependencies
- [x] Update Node to LTS 14

### CODE REVIEW

**Comment by SimonKindhauser** [comment]
Update Node JS version in set-up documentation

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "chexxo-server",
-  "version": "0.0.1",
+  "version": "3.1.0",
```
Comment (Vec7or) [inline package.json:3]: Please also adjust the version in the OpenApiDoc

Comment (Vec7or) [inline package.json:3]: Please also adjust the version in the openApiDoc

File: package.json [hunk 2]
```diff
@@ -31,20 +31,20 @@
...
-    "prettier": "2.2.1",
+    "prettier": "^2.2.1",
     "ts-jest": "^26.4.4",
-    "ts-loader": "^8.0.11",
+    "ts-loader": "^9.1.2",
     "typedoc": "^0.19.2",
     "typedoc-plugin-external-module-name": "^4.0.3",
     "typedoc-plugin-no-inherit": "^1.2.0",
-    "typescript": "^4.1.2",
+    "typescript": "^4.0.2",
```
Comment (Vec7or) [inline package.json:47]: Why a downgrade?

File: src/api/AWSAPIProvider.test.ts [hunk 3]
```diff
@@ -23,20 +30,6 @@ beforeEach(() => {
   apiProvider.init(certificateProvider, null);
 });
 
-test("Unsupported endpoint", () => {
```
Comment (Vec7or) [inline src/api/AWSAPIProvider.test.ts:26]: Please replace this tests with new ones compatible with the new event.

File: README.md [hunk 4]
```diff
@@ -32,36 +32,37 @@ In order to deploy the express variant of the server these steps have to be foll
...
-13. Change `ANY` to `GET` and click `Save`.
-14. On the left hand side of the gateway-configuration select `Integrations`.
+13. Change `ANY` to `GET`, the path to `/certificate/{domain+}`, and click `Save`.  
+![Change Route](img/13-change-route.jpg)
+14. On the left hand side of the gateway-configuration select `Integrations`.  
 ![Select Integrations](img/11-select-integrations.jpg)
-15. Select `$default` and click on `Edit Integration`.
+15. Select `$default` and click on `Edit Integration`.  
 ![Select default](img/12-select-default.jpg)
 16. Make sure your function name is inside the `Lambda function` field and on `Save`. It is important, that you click on `Save` even if the function name was already correct.
```
Comment (Vec7or) [inline README.md:63]: This does no longer work or does it?

**Review by Vec7or — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (SimonKindhauser) [reply package.json:47]: version conflict, with typedoc

File: src/api/AWSAPIProvider.test.ts (hunk 3 above)
Comment (SimonKindhauser) [reply src/api/AWSAPIProvider.test.ts:26]: there are no replacements since this functionality is now handled by aws.

File: README.md (hunk 4 above)
Comment (SimonKindhauser) [reply README.md:63]: The button is gone, but I don't know what the purpose was.. should I just delete the related lines?

Comment (Vec7or) [reply README.md:63]: If it is no longer necessary you may delete the corresponding lines.

File: README.md [hunk 5]
```diff
@@ -32,36 +32,35 @@ In order to deploy the express variant of the server these steps have to be foll
...
-13. Change `ANY` to `GET` and click `Save`.
-14. On the left hand side of the gateway-configuration select `Integrations`.
+13. Change `ANY` to `GET`, the path to `/certificate/{domain+}`, and click `Save`.  
+![Change Route](img/13-change-route.jpg)
+14. On the left hand side of the gateway-configuration select `Integrations`.  
 ![Select Integrations](img/11-select-integrations.jpg)
-15. Select `$default` and click on `Edit Integration`.
-![Select default](img/12-select-default.jpg)
-16. Make sure your function name is inside the `Lambda function` field and on `Save`. It is important, that you click on `Save` even if the function name was already correct.
+16. Make sure your function name is inside the `Lambda function` field.
```
Comment (Vec7or) [inline README.md:61]: Wrong number

**Review by Vec7or — CHANGES_REQUESTED** [review]

**Review by Vec7or — APPROVED** [review]

**Pull request merged**